setup.py

```diff
@@ -51,7 +51,7 @@
         (BASE_DIR, ['data/nssm_original.exe'])
     )],
     install_requires=['indy-plenum==1.9.0.dev847',
-                      'python-dateutil',
+                      'python-dateutil==2.6.1',
```

Comment:
I think this dependency can be removed at all since plenum already depends on this version of python-dateutil